app/controllers/organisers_controller.rb

```diff
@@ -55,6 +55,11 @@ def destroy
     @organizer.delete
     redirect_to :back, notice: "Deleted successfully!"
   end
+  
+  def problematic_projects
+    @projects = Project.all
+    render "problematic_projects"
```

Comment:
@pg20 if the view has the same name as the method you don't need to use render here